### PR TITLE
Add Native prefix (LocalDate)

### PR DESCRIPTION
**Pull request by solodkiy**
LocalDate::fromNativeDateTime
LocalDate::toNativeDateTime
LocalDate::toNativeDateTimeImmutable

https://github.com/brick/date-time/issues/47

### CODE REVIEW

File: src/LocalDate.php [hunk 1]
```diff
@@ -711,10 +731,21 @@ public function toDateTime() : \DateTime
      * Converts this LocalDate to a native DateTimeImmutable object.
      *
      * The result is a DateTimeImmutable with time 00:00 in the UTC time-zone.
+     * @deprecated please use toNativeDateTime instead
```
Comment (BenMorel) [inline src/LocalDate.php:734]: Should be toNativeDateTimeImmutable

**Comment by BenMorel** [comment]
Missing CHANGELOG entry as well!

**Comment by solodkiy** [comment]
@BenMorel fixed

**Comment by BenMorel** [comment]
Thank you, @solodkiy !

**Pull request merged**